src/Library/demos/Color Picker/main.js

```diff
@@ -0,0 +1,26 @@
+import Gtk from "gi://Gtk";
+import Gdk from "gi://Gdk";
+import Gio from "gi://Gio";
+import GLib from "gi://GLib";
+import Xdp from "gi://Xdp";
+import XdpGtk from "gi://XdpGtk4";
+
+const portal = new Xdp.Portal();
+const parent = XdpGtk.parent_new_gtk(workbench.window);
+const button = workbench.builder.get_object("button");
+
+Gio._promisify(Xdp.Portal.prototype, "pick_color", "pick_color_finish");
+
+async function pickColor() {
+  try {
+    const color = await portal.pick_color(parent, null);
```

Comment:
the result of `pick_color` isn't really a color, I would naming the return value `result` instead and move the comment there

```suggestion
    // result is a GVariant of the form (ddd), containing red, green and blue components in the range [0,1]
    const result = await portal.pick_color(parent, null);
```